applications/app/views/crossword.scala.html

```diff
@@ -43,9 +43,10 @@ <h3 class="crossword__clues-header">Across</h3>
 
                                     <div class="crossword__clues--down">
                                         <h3 class="crossword__clues-header">Down</h3>
-                                        @crosswordEntries(crosswordPage.crossword.entries.filter(_.direction == Down))
+                                        <noscript>@crosswordEntries(crosswordPage.crossword.entries.filter(_.direction == Down))
```

Comment:
this means the crossword grid and clues are rendered if you don't run the js app. the js app take a short while to get going, when it does, it re-draws the whole thing. if we show it before the app, it means we have 2 templates to keep in sync, but more importantly you have an unresponsive crossword that suddenly flickers and then becomes responsive.  idea was just to show a placeholder, fb-style, outside `noscript`.